xbmc/video/VideoInfoScanner.cpp

```diff
@@ -818,6 +886,72 @@ namespace VIDEO
     return INFO_NOT_FOUND;
   }
 
+  bool CVideoInfoScanner::UpdateSetInfo(CFileItem* pItem)
+  {
+    // If there is a set, see if the details need to be updated from the
+    // Movie Set Information Folder
+    bool setUpdated = false;
+    if (!pItem->GetVideoInfoTag()->m_set.title.empty())
+    {
+      const std::string movieSetInfoPath =
+          VIDEO::CVideoInfoScanner::GetMovieSetInfoFolder(pItem->GetVideoInfoTag()->m_set.title);
+      if (!movieSetInfoPath.empty())
+      {
+        // look for Set.NFO
+        std::unique_ptr<ISetInfoTagLoader> setLoader;
+        AddonPtr addon{};
+        CServiceBroker::GetAddonMgr().GetAddon("metadata.local", addon, ADDON::OnlyEnabled::CHOICE_YES);
+        ScraperPtr scraper = std::dynamic_pointer_cast<CScraper>(addon);
+        setLoader.reset(CSetInfoTagLoaderFactory::CreateLoader(*pItem, scraper));
+        if (setLoader)
+        {
+          CSetInfoTag s{};
+          CInfoScanner::INFO_TYPE result{CInfoScanner::NO_NFO};
+          result = setLoader->Load(s, false);
+          if (!s.IsEmpty())
+          {
+            if (!s.GetTitle().empty())
+              pItem->GetVideoInfoTag()->m_set.title = s.GetTitle();
+            if (!s.GetOverview().empty())
+              pItem->GetVideoInfoTag()->m_set.overview = s.GetOverview();
+            setUpdated = true;
+          }
+        }
+
+        // Now look for art
+        CGUIListItem::ArtMap movieSetArt;
+        const std::vector<std::string> movieSetArtTypes =
+            CVideoThumbLoader::GetArtTypes(MediaTypeVideoCollection);
+        CVideoInfoScanner::AddLocalItemArtwork(movieSetArt, movieSetArtTypes, movieSetInfoPath,
+                                               true, false);
+        if (!movieSetArt.empty())
+        {
+          // Remove existing set art from VideoInfoTag
+          pItem->GetVideoInfoTag()->m_strPictureURL.Parse();
+          auto urls = pItem->GetVideoInfoTag()->m_strPictureURL.GetUrls();
+          urls.erase(std::remove_if(urls.begin(), urls.end(),
+                                    [&](const auto& url)
```

Comment:
No need to capture any vars here. I think you can just remove the `&` and end up with `[]'.